### PR TITLE
Useful wrappers and helpers

### PR DESCRIPTION
**Pull request by vg-svitla**
1) aio helpers
2) logging helpers

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -51,6 +51,19 @@ pylint = "*"
 pytest = "*"
 pytest-cov = "*"
 requests-mock = "^1.9"
+aiohttp = "^3.8.4"
+aiofiles = "^23.1.0"
+aiocsv = "^1.2.4"
+loguru = "^0.7.0"
+aiolimiter = "^1.1.0"
+aiobotocore = "^2.5.2"
```
Comment (squioc) [inline pyproject.toml:59]: I think that these dependencies should be in the `tool.poetry.dependencies` section.
I suggest setting these dependencies as optional and use the [`extra`](https://python-poetry.org/docs/pyproject/#extras) to allow customers to install these dependencies only when needed

File: sekoia_automation/helpers/aio/__init__.py
Comment (squioc) [inline sekoia_automation/helpers/aio/__init__.py:1]: I suggest reverse `aio` and `helper`, and replace `aio` with `async` in the path, ie: sekoia_automation/async/helpers/__init__.py
In order to create an async section in the sdk.

**Review by squioc — CHANGES_REQUESTED** [review]
Nice work.
Could you add an entry in the Unreleased part on the CHANGELOG?

File: sekoia_automation/helpers/aio/__init__.py
Comment (vg-svitla) [reply sekoia_automation/helpers/aio/__init__.py:1]: Good point, but we cannot rename to async, as it is reserved keyword
So I passed `sekoia_automation/aio/helpers`
Looking forward for suggestions :)

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -51,12 +51,12 @@ jobs:
 
       - name: Install dependencies
         id: install-dependencies
-        run: poetry install
+        run: poetry install --extras=all
 
       - name: Execute Python tests
         id: execute-tests
         run: |
-          poetry run python -m pytest --junit-xml=junit.xml --cov-report term --cov-report xml:coverage.xml --cov=sekoia_automation tests
```
Comment (squioc) [inline .github/workflows/ci.yml:59]: Why did you remove the directory `tests` in the command line?

File: .pre-commit-config.yaml
Comment (squioc) [inline .pre-commit-config.yaml:1]: @Darkheir Do you agree to this change?

File: sekoia_automation/aio/connector.py
Comment (squioc) [inline sekoia_automation/aio/connector.py:1]: :+1:

**Review by squioc — COMMENTED** [review]
Nice job. I love the async connector.

@darkheir Could you validate some points? Thanks

File: .github/workflows/ci.yml (hunk 2 above)
Comment (vg-svitla) [reply .github/workflows/ci.yml:59]: I removed it because of incorrect configuration between local execution and gitlab ci.
For some reason it does not resolve paths with `tests` on local env

File: .pre-commit-config.yaml
Comment (Darkheir) [reply .pre-commit-config.yaml:1]: Fine by me if it works. It will allow to run checks with the same version we will have in the CI pipeline.

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -1,22 +1,25 @@
...
+          entry: poetry run ruff
+          language: system
+          types: [ python ]
+          args: [ --fix, --exit-non-zero-on-fix, . ]
 
-    - repo: https://github.com/pre-commit/mirrors-mypy
-      rev: 'v1.3.0'  # Use the sha / tag you want to point at
-      hooks:
         - id: mypy
-          args: [--install-types, --non-interactive]
```
Comment (Darkheir) [inline .pre-commit-config.yaml:21]: Why remove `--install-types` ?

Comment (vg-svitla) [reply .pre-commit-config.yaml:21]: I moved all mypy config to poetry.yaml so no need to add additional args here

**Review by squioc — APPROVED** [review]
